Add feature to add all events from a race week to Apple Calendar app

qeude opened the pull request:
## Description

I added a feature to add all events from a race week to the Apple Calendar application. It basically uses AppleScript to access the Mac application, retrieve calendars, and add events to the selected calendar.

## Screencast
![Screen Recording 2022-04-23 at 00 54 45](https://user-images.githubusercontent.com/19294297/164815016-d0f045b8-6f24-41f2-ae1b-66f9c1c839f3.gif)

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

sxn commented:
hi @qeude - nice changes! Hope you don't mind, I've sneaked a small change in as well 😅 

A couple of mentions:
1. Could you update the `CHANGELOG.md` file to reflect this change?
2. Trying to add the sessions to a calendar I'm not allowed to write to results in the extension crashing:

https://user-images.githubusercontent.com/1155589/165083569-0f5f4fe9-3946-4d7a-b477-e1681cddaea4.mov

Let me know once you've had a chance to fix the issue or if you need some help either here or on [Slack](https://raycastcommunity.slack.com/) 😸

sxn commented on extensions/f1-standings/src/views/AddToCalendar.tsx:
```diff
@@ -0,0 +1,88 @@
+            const endDate = moment(date[1]).add(2, "hours");
+            return `make new event with properties { summary:"🏎 ${getFlag(race.Circuit.Location.country)} ${
+              date[0]
+            } - ${race.raceName}", start date:(date "${moment(date[1]).format(
```
is `moment` necessary here? The extension already has a [formatDate](https://github.com/raycast/extensions/blob/main/extensions/f1-standings/src/utils/formatDate.ts) function which takes into account the user's locale 🤔 can'd we use that instead?

qeude replied:
I did use `moment` so I can format to the specific needed format for AppleScript. IMO `moment` makes it much smoother in that case. 🤷

qeude commented:
I'm going to take a look at these feedbacks ASAP 🙌🏼

qeude commented:
@sxn No worries about adding some changes, it's always ok to improve an extension together 😄 
I've made the changes you requested:
- Changed the `CHANGELOG.md`.
- Changed the used AppleScript so it doesn't list calendars that are not writable.
- Didn't change [`moment.js` for the reason explained here](https://github.com/raycast/extensions/pull/1470#discussion_r857611893). Even if I tend to be a "no-third" dependency guy, I feel that `moment.js` is pretty mature to be used and useful for readability here.

sxn commented:
nice! 🎉

csigritz commented:
thx guys...
